spec/system/placing_an_order_on_behalf_of_spec.rb

```diff
@@ -84,10 +84,11 @@
     choose account.to_s
     click_button "Continue"
 
-    find("label", text: /More options/i).click
     fill_in "Order date", with: I18n.l(2.days.ago.to_date, format: :usa)
     select "Complete", from: "Order Status"
 
+    fill_in "Fulfilled at", with: I18n.l(2.days.ago.to_date, format: :usa)
```

Comment:
So `I18n.l(2.days.ago.to_date, format: :usa) => "08/06/2022"` and `I18n.l(2.days.ago.beginning_of_day, format: :usa) => "08/06/2022 12:00 AM"`. Since this is used to fill out a date, looks like `to_date` actually does need to be used here.